src/app/data-planes/data/index.ts

```diff
@@ -13,56 +17,138 @@ export type DataplaneWarning = {
   }
 }
 
-export function getLastUpdateTime(dataplaneOverview: DataplaneOverview): string | undefined {
-  const subscriptions = dataplaneOverview.dataplaneInsight?.subscriptions ?? []
-  if (subscriptions.length === 0) {
-    return undefined
+export type DataplaneNetworking = PartialDataplaneNetworking
+
+export type Dataplane = PartialDataplane & {
+  networking: DataplaneNetworking
+}
+
+export type DataplaneInsight = PartialDataplaneInsight & {
+  subscriptions: DiscoverySubscription[]
+  connectedSubscription?: DiscoverySubscription
+}
+
+export type DataplaneOverview = PartialDataplaneOverview & {
+  dataplane: {
+    networking: DataplaneNetworking
   }
+  dataplaneInsight: DataplaneInsight
+  dataplaneType: 'standard' | 'builtin' | 'delegated'
+  status: 'online' | 'offline' | 'partially_degraded'
+  unhealthyInbounds: Array<{ service: string, port: number }>
+  lastUpdateTime?: string | undefined
+  warnings: DataplaneWarning[]
+  isCertExpired: boolean
+  serviceTags: LabelValue[]
```

Comment:
Should this just be `services`?

Reply:
Hm, kinda nice to know you’re not looking at a list of service names, but it doesn’t matter much. I changed it.

Reply:
Oh sorry I meant that this should be an array of services, not tags. Where we use this its not that we want tags its that we want a list if service names right? (or did I misunderstand this?)

Reply:
Oh, I see. Sure, that makes sense as well. I changed this to be a `string[]`.